From f0f96eecf9d3fa9ca8dbdf1c03b5f323a74a009c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <baimingcong@uniontech.com>
Date: Fri, 28 Jun 2024 16:23:20 +0800
Subject: [PATCH 038/101] net: ethernet: phytium: add a missing declaration for
 *np
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix a compilation error:

drivers/net/ethernet/phytium/phytmac_platform.c: In function ‘phytmac_plat_probe’:
drivers/net/ethernet/phytium/phytmac_platform.c:98:55: error: ‘np’ undeclared (first use in this function); did you mean ‘up’?
   98 |                 match = of_match_node(phytmac_dt_ids, np);
      |                                                       ^~
      |                                                       up

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>
Ref: https://github.com/deepin-community/kernel-rolling/commit/eec8d65bd08c528c7909a78e1a25b0cbcff9cadc
---
 drivers/net/ethernet/phytium/phytmac_platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/phytium/phytmac_platform.c b/drivers/net/ethernet/phytium/phytmac_platform.c
index 797909211fed1..343e1a5ce564a 100644
--- a/drivers/net/ethernet/phytium/phytmac_platform.c
+++ b/drivers/net/ethernet/phytium/phytmac_platform.c
@@ -77,6 +77,7 @@ static int phytmac_get_phy_mode(struct platform_device *pdev)
 static int phytmac_plat_probe(struct platform_device *pdev)
 {
 	const struct phytmac_config *phytmac_config = &phytium_1p0_config;
+	struct device_node *np;
 	struct resource *regs;
 	struct phytmac *pdata;
 	int ret, i;
-- 
2.46.0

